lib/ansible/modules/cloud/amazon/s3.py

```diff
@@ -129,10 +132,9 @@
     version_added: "1.2"
   region:
     description:
-     - >
-       AWS region to create the bucket in. If not set then the value of the AWS_REGION and EC2_REGION environment variables are checked,
-       followed by the aws_region and ec2_region settings in the Boto config file.  If none of those are set the region defaults to the
-       S3 Location: US Standard.  Prior to ansible 1.8 this parameter could be specified but had no effect.
+     - "AWS region to create the bucket in. If not set then the value of the AWS_REGION and EC2_REGION environment variables
+       are checked, followed by the aws_region and ec2_region settings in the Boto config file. If none of those are set the
+       region defaults to the S3 Location: US Standard. Prior to ansible 1.8 this parameter could be specified but had no effect."
     required: false
```

Comment:
I'm thrilled to have another set of eyes on this. I think you're right. When I was working on this I couldn't actually find Location.Default in Boto3.  And since this module needs to be backwards compatibility I think I'll just keep the default hard coded to us-east-1 like is happening now
https://github.com/s-hertel/ansible/blob/8f44e7482c2aa9e31139e1a915d016587009aefa/lib/ansible/modules/cloud/amazon/s3.py#L584-L592. When I come back to this to tidy it up again I'll be sure to fix up the docs and remove any other traces of Default.Location :) Good catch!